packages/ladle/lib/shared/types.ts

```diff
@@ -186,6 +186,7 @@ export type Config = {
   outDir: string;
   base?: string;
   mode?: string;
+  noWatch?: boolean;
```

Comment:
```suggestion
  noWatch: boolean;
```

and adding

```
noWatch: false
```

here https://github.com/tajo/ladle/blob/main/packages/ladle/lib/shared/default-config.js#L4